src/turner_lab_to_nwb/asap_tdt/asap_tdtnwbconverter.py

```diff
@@ -1,33 +1,15 @@
 from typing import Optional
 
 import numpy as np
-import pandas as pd
-from neuroconv import NWBConverter
-from neuroconv.datainterfaces import PlexonSortingInterface
-from neuroconv.tools.nwb_helpers import get_default_backend_configuration, configure_backend
+from neuroconv import ConverterPipe
 from neuroconv.utils import DeepDict
 from pynwb import NWBFile
 
-from turner_lab_to_nwb.asap_tdt.interfaces import (
-    ASAPTdtEventsInterface,
-    ASAPTdtFilteredRecordingInterface,
-    ASAPTdtRecordingInterface,
-    ASAPTdtSortingInterface,
-)
 
-
-class AsapTdtNWBConverter(NWBConverter):
-    """Primary conversion class for Turner's extracellular electrophysiology dataset."""
-
-    data_interface_classes = dict(
-        Recording=ASAPTdtRecordingInterface,
-        ProcessedRecordingVL=ASAPTdtFilteredRecordingInterface,
-        ProcessedRecordingGPi=ASAPTdtFilteredRecordingInterface,
-        PlexonSortingVL=PlexonSortingInterface,
-        PlexonSortingGPi=PlexonSortingInterface,
-        CuratedSorting=ASAPTdtSortingInterface,
-        Events=ASAPTdtEventsInterface,
-    )
+class ASAPTdtNWBConverter(ConverterPipe):
```

Comment:
The reason for changing to `ConverterPipe` is because we don't know in advance whether we have a single .flt.mat or .plx file, and whether we have curated units at all in the .mat events file before looking at the files we have.